yoti-sdk-sandbox/src/main/java/com/yoti/api/client/sandbox/docs/request/task/SandboxDocumentTextDataExtractionTask.java

```diff
@@ -20,9 +20,13 @@ public class SandboxDocumentTextDataExtractionTask {
     @JsonProperty("document_filter")
     private final SandboxDocumentFilter documentFilter;
 
-    SandboxDocumentTextDataExtractionTask(SandboxDocumentTextDataExtractionTaskResult result, SandboxDocumentFilter documentFilter) {
+    @JsonProperty("response_delay")
+    private final Integer responseDelay;
+
+    SandboxDocumentTextDataExtractionTask(SandboxDocumentTextDataExtractionTaskResult result, SandboxDocumentFilter documentFilter, Integer responseDelay) {
```

Comment:
Shouldn't we have a common base class for these, to avoid duplicating everything...?